tests/test_user.py

```diff
@@ -990,6 +990,7 @@ def test_merge(self):
                                   last_name='Better')
             other.birthdate = '02-05-1968'
             other.gender = 'male'
+            other.timezone = 'US/Eastern'
```

Comment:
Should this be `America/New_York`? I think that's the standardized name...

Reply:
Never mind, they are [two distinct values](https://stackoverflow.com/questions/9863625/difference-between-est-and-america-new-york-time-zones/9863679#9863679):

>EST is UTC - 5 hours. America/New_York is EST in the winter and E*D*T in the summer

Reply:
I'll note, those are simply testing values to confirm we keep the one we want.